semperfiwebdesign/all-in-one-seo-pack
# Removes legacy code for Google Connect option

Issue #2395

## Proposed changes
This removes the option and help code associated with the old Google Connect button and removes the extlib directory and OAuth.php file associated with this option.

## Types of changes

What types of changes does your code introduce?
- Removing old code/functionality

## Checklist

## Testing instructions
- Verify that all options associated with Google Analytics still work and that the correct GA tracking code is output.  
- Verify that there are no errors associated with the removed OAuth.php file.

## Further comments
Please verify that there is no other code that calls inc/extlib/OAuth.php.
